standalone/instructlab-rhoai.md

```diff
@@ -43,9 +43,12 @@ Before running the training and evaluation steps we must complete the following:
   ```
 
 * Download ilab model repository in s3-data model direct
+> [!NOTE]
+> [skopeo](https://github.com/containers/skopeo/) must be installed and policy.json file present in `$HOME/.config/containers/policy.json` or `/etc/containers/policy.json`
+> If you don't have skopeo policy, you can download default policy from [here](https://github.com/containers/skopeo/blob/main/default-policy.json)
```

Comment:
Do you think this should point to a RH doc instead?

Reply:
do you mean something like https://github.com/containers/skopeo/blob/main/install.md ? I am not sure if we have red hat specific docs for installing skopeo?

Reply:
How about https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/building_running_and_managing_containers/assembly_running-skopeo-buildah-and-podman-in-a-container#proc_running-skopeo-in-a-container_assembly_running-skopeo-buildah-and-podman-in-a-container?

Reply:
@leseb I think `ilab model download` can't call skopeo from within a container. it calls it using subprocess https://github.com/instructlab/instructlab/blob/0a8dee3857f9e5e2ce9cee8691033ad9059eef34/src/instructlab/model/download.py#L212-L224 

the only available guide is https://github.com/containers/skopeo/blob/main/install.md which you can find it in the repository.

Reply:
Right, I missed it was running inside a container. I guess my point is that since this is a product doc we shouldn't point to an upstream installation guide.

Reply:
@HumairAK this is correct,I thought it was phase 2 and related to the earlier concern we had. I agree that since this i all running from a RHELAI box then skopeo is already available. No need to mention this IMO.

Reply:
done!